src/DonationForms/ListTable/Columns/LevelsColumn.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\DonationForms\ListTable\Columns;
+
+use Give\DonationForms\Models\DonationForm;
+use Give\Framework\ListTable\ModelColumn;
+
+/**
+ * @unreleased
+ *
+ * @extends ModelColumn<DonationForm>
+ */
+class LevelsColumn extends ModelColumn
+{
+
+    /**
+     * @unreleased
+     *
+     * @inheritDoc
+     */
+    public static function getId(): string
+    {
+        return 'levels';
+    }
+
+    /**
+     * @unreleased
+     *
+     * @inheritDoc
+     */
+    public function getLabel(): string
+    {
+        return __('Donation Levels', 'give');
+    }
+
+    /**
+     * @unreleased
+     *
+     * @inheritDoc
+     *
+     * @param DonationForm $model
+     */
+    public function getCellValue($model, $locale = ''): string
```

Comment:
Is there any reason to set the default of `$locale` here as an empty string? I thought the value was set through the `items()` method from the request.

Reply:
Paulo confirmed we cannot because this is an abstract method.  We can disregard for now.